hack/update-swagger-spec.sh

```diff
@@ -18,7 +18,7 @@ function cleanup()
     rm -rf "${TEMP_DIR}"
 
     if [ $out -ne 0 ]; then
-        echo "[FAIL] !!!!! Generate Failed !!!!"
+        echo -e "\033[0;31m [FAIL] !!!!! Generate Failed !!!!\033[0m"
```

Comment:
It is a feature I really enjoyed from Kube's verification scripts, so I added it. Does anyone ever pipe verification script output? To what? It seems like if it impeded workflow it would be gone from their repo as well. I can remove it or add pipe logic if you think it's necessary, but at the moment I'm unconvinced.